l1-contracts/src/governance/proposer/EmpireBase.sol

```diff
@@ -102,7 +99,7 @@ contract GovernanceProposer is IGovernanceProposer {
    */
   function pushProposal(uint256 _roundNumber) external override(IGovernanceProposer) returns (bool) {
```

Comment:
nit: Maybe this function name needs to change. Certainly the @notice above needs to.